bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ioos-us",
+  "version": "0.0.1",
+  "homepage": "https://github.com/ioos/ioos-us",
+  "authors": [
+    "Luke Campbell <luke.s.campbell@gmail.com>"
+  ],
+  "description": "The homepage for http://ioos.us/",
+  "main": "",
```

Comment:
can you update the metadata above?

Reply:
Lock down all versions so no ^ carrot? In npm and bower?

Reply:
Preference on package.json name. Currently name: "ioos-us"

Reply:
comt-landing